caluma/settings/django.py

```diff
@@ -156,3 +156,11 @@ def parse_admins(admins):  # pragma: no cover
 # GraphQL
 if DEBUG:
     GRAPHENE["MIDDLEWARE"].append("graphene_django.debug.DjangoDebugMiddleware")
+
+
+# caluma access log
+ENABLE_ACCESS_LOG = env.bool("ENABLE_ACCESS_LOG", default=False)
+
+if ENABLE_ACCESS_LOG:
```

Comment:
I'd trigger only the logging, not enabling / disabling the whole app. Otherwise you might run into problems if after the config change `migrate` is not being run

Reply:
That's true, having the app in INSTALLED_APPS shouldn't hurt anyways.